README.md

```diff
@@ -1 +1,8 @@
-# node-fn-structured-cloudevent-buildpack
+# `node-fn-structured-cloudevent-buildpack`
```

Comment:
nitpick: can we give it a human readable name like (takes a deep breath) "Heroku Buildpack for Structured Cloudevents in Node Functiosn" or something?

Reply:
"Node Function Structured Cloudevent Middleware" from buildpack.toml would work